index.html

```diff
@@ -41,60 +35,161 @@ <h2 class="primary_color">I'm a software developer</h2>
-          I can help you build a product , feature or website Look
-          through some of my work and experience! <br />If you like
-          what you see and have a project you need coded, don't
+          I can help you build a product , feature or website Look through some of my work and
+          experience! <br />If you like what you see and have a project you need coded, don't
           hestiate to contact me.
         </p>
       </div>
 
       <ul class="social_icons_container">
         <li>
           <a href="https://twitter.com/marcealarconb">
-            <img
-              class="social-icon"
-              src="images/twitter.svg"
-              alt="Twitter contact"
-            />
+            <img class="social-icon" src="images/twitter.svg" alt="Twitter contact" />
           </a>
         </li>
         <li>
           <a href="https://www.linkedin.com/in/marceloalarconb/">
-            <img
-              class="social-icon"
-              src="images/linkedin.svg"
-              alt="Linkedin contact"
-            />
+            <img class="social-icon" src="images/linkedin.svg" alt="Linkedin contact" />
           </a>
         </li>
         <li>
           <a href="https://medium.com/@marceloalarconbarrenechea">
-            <img
-              class="social-icon"
-              src="images/medium.svg"
-              alt="Medium contact"
-            />
+            <img class="social-icon" src="images/medium.svg" alt="Medium contact" />
           </a>
         </li>
         <li>
           <a href="https://github.com/chelobotix/">
-            <img
-              class="social-icon"
-              src="images/github.svg"
-              alt="GitHub contact"
-            />
+            <img class="social-icon" src="images/github.svg" alt="GitHub contact" />
           </a>
         </li>
         <li>
           <a href="https://angel.co/u/marcelo-alarcon-1">
-            <img
-              class="social-icon"
-              src="images/angellist.svg"
-              alt="Angelist contact"
-            />
+            <img class="social-icon" src="images/angellist.svg" alt="Angelist contact" />
           </a>
         </li>
       </ul>
     </section>
+
+    <div class="d-flex justify-content-center">
+      <img src="images/arrow.png" alt="down arrow" />
+    </div>
+
+    <!-- Works Section -->
+
+    <section class="grid_5x1">
+      <div>
+        <h2 class="secondary_color">My Recent <br />Works</h2>
+      </div>
+
+      <article class="grid_5x1">
+        <a href="#">
+          <img src="images/Snapshoot Portfolio.png" alt="Tonic Website Snapshoot" />
+        </a>
+        <h3 class="tertiary_color">Tonic</h3>
+        <ul class="d-flex">
+          <li class="li_works tertiary_color">CANOPY</li>
+          <li class="li_works tertiary_color">css</li>
+          <li class="li_works tertiary_color">Javascript</li>
+        </ul>
+        <a class="btn1" href="#">See Projects</a>
```

Comment:
- [ ] Kindly add a hover and pressed state to your buttons to match the figma template. You can do this using the `:hover` and `:active` pseudoclasses.
![btn3](https://user-images.githubusercontent.com/94551484/205328805-b74b297b-9742-434d-8fdc-9c986c0202d5.png)

Reply:
:red_circle: I think the state of the buttons should be changing depending on the action. Kindly visit these links to know more about the implementation ( [hover](https://developer.mozilla.org/en-US/docs/Web/CSS/:hover), [enabled](https://developer.mozilla.org/en-US/docs/Web/CSS/:enabled) )

<img src="https://user-images.githubusercontent.com/94551484/205328805-b74b297b-9742-434d-8fdc-9c986c0202d5.png">